var/spack/repos/builtin/packages/iq-tree/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class IqTree(CMakePackage):
+    """IQ-TREE Efficient software for phylogenomic inference"""
+
+    homepage = "http://www.iqtree.org"
+    git      = "https://github.com/Cibiv/IQ-TREE.git"
+    url      = "https://github.com/Cibiv/IQ-TREE/archive/v1.6.12.tar.gz"
+
+    version('1.6.12',   sha256='9614092de7a157de82c9cc402b19cc8bfa0cb0ffc93b91817875c2b4bb46a284') 
+
+    variant('openmp', default=True, description='Enable OpenMP support.')
+    variant('mpi',    default=False, description='Enable MPI support.')                                                                                                   
+
+    maintainers = ['ilbiondo']
+
+    # Depends on Eigen3 and zlib
+
+    depends_on("boost")
+    depends_on("eigen")
+    depends_on("zlib")
+    depends_on('mpi', when='+mpi')
+
+    def cmake_args(self):
+
+        spec = self.spec
+        args = []
+        iqflags = []
+
+        if '+openmp' in spec:
+            iqflags.append('omp')
+
+        if '+mpi' in spec:
+            iqflags.append('mpi')
+
+        args.append('-DIQTREE_FLAGS=' + ",".join([str(f) for f in iqflags]))
```

Comment:
Would this:
```suggestion
        args.append('-DIQTREE_FLAGS=' + ",".join(iqflags))
```
work? Also, what happens if `~openmp~mpi` and `iqflags` is empty?